### PR TITLE
Manage mobile bugfix

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

A provider has reported that the ProviderInterface is unusable on mobile viewports.

## Changes proposed in this pull request

Because the position is fixed, when on mobile or tablet the filter content is not displayed in the page.
By transforming into position relative the page worked.

## Guidance to review

1. Does it work?
2. Does it affects the other pages?

## Link to Trello card

https://trello.com/c/NfwQ20pI/1475-bug-providerinterface-mobile-is-unusable

### CODE REVIEW

File: app/frontend/styles/components/_paginated_filter.scss [hunk 1]
```diff
@@ -80,6 +80,17 @@
...
+// Fix for fixed position of layout filters
+.js-enabled .moj-filter-layout__filter {
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+  }
+
+  @include govuk-media-query($from: mobile) {
+    position: relative;
+  }
+}
```
Comment (tomas-stefano) [inline app/frontend/styles/components/_paginated_filter.scss:92]: That was the only way I saw to override the .moj-filter-layout__filter

Other solution was to use:

```
.app-filter {
@include govuk-media-query($from: tablet) {
    position: relative !important;
  }

  @include govuk-media-query($from: mobile) {
    position: relative !important;
  }
}
```

If you know other way, I am happy to change!

File: app/frontend/styles/components/_paginated_filter.scss [hunk 2]
```diff
@@ -80,6 +80,17 @@
   }
 }
 
+// Fix for fixed position of layout filters
+.js-enabled .moj-filter-layout__filter {
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+  }
```
Comment (dcyoung-dev) [inline app/frontend/styles/components/_paginated_filter.scss:87]: Do we need to do this for tablet? I haven't managed to see an issue on tablet sizes.

**Review by dcyoung-dev — APPROVED** [review]
I prefer the `position: relative !important;` option as it smells more and may lead us back here.
Looking at the behaviour of these filters compared to the filters on Find - it looks like we could be doing with a bit of design - Find has a "show filters" button which would work well on this page 🎨

For now this is good fix - could we create a card for a bit of design work to long term fix this

File: app/frontend/styles/components/_paginated_filter.scss (hunk 2 above)
Comment (tomas-stefano) [reply app/frontend/styles/components/_paginated_filter.scss:87]: On my local there were some problems on iPad mini and other small tablets formats.

**Review by elceebee — APPROVED** [review]
This looks good to me! 

There are some odd things happening with the 'Filter' heading and the width of the 'Apply filters' button. It appears to have a different break point than the filter box itself.  

This might be beyond the scope of this ticket, and I'm definitely not blocking it over this. But if it's an easy fix, might as well make it consistent.

image

image

**Pull request merged**